src/utils/string.test.ts

```diff
@@ -1,6 +1,24 @@
 import { describe, expect, it } from 'vitest';
 
-import { toBoolean, toFloat } from './string';
+import { isBlank, toBoolean, toFloat } from './string';
+
+describe('isBlamk', () => {
```

Comment:
`blaMk` 😄

Reply:
this is what happens when you type while holding a kid :rofl: